source/stylesheets/screen.css.scss

```diff
@@ -2,6 +2,8 @@
 @import 'normalize';
 @import 'variables';
 @import 'icon-font';
+@import 'font-awesome-sprockets';
+@import 'font-awesome';
```

Comment:
Honestly, I think somebody else did the font file generation — if you're willing to do SVG, I think that would be much preferred!